ivy_tests/test_ivy/test_frontends/test_jax/test_jax_random.py

```diff
@@ -122,3 +122,74 @@ def call():
     for (u, v) in zip(ret_np, ret_from_np):
         assert u.dtype == v.dtype
         assert u.shape == v.shape
+
+
+@st.composite
+def _get_all_beta_param(draw):
+    dtype_key, key = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["uint32"],
+            min_value=0,
+            max_value=2000,
+            min_num_dims=1,
+            max_num_dims=1,
+            min_dim_size=2,
+            max_dim_size=2,
+        )
+    )
+    alpha = draw(st.floats(min_value=0, max_value=5, exclude_min=True))
+    beta = draw(st.floats(min_value=0, max_value=5, exclude_min=True))
+    shape = draw(
+        helpers.get_shape(
+            min_num_dims=2, max_num_dims=2, min_dim_size=1, max_dim_size=5
+        )
+    )
+    dtype = draw(
+        helpers.get_dtypes("float", full=False).filter(
+            lambda x: "float16" not in x[0] and "bfloat16" not in x[0]
+        )
+    )
+    return dtype_key, key, alpha, beta, shape, dtype
```

Comment:
No need to make a composite function for all this, you could shove it in the `handle_frontend_test`, you only need to use this when one generated value depends on another, here none do